src/ServiceControl.Monitoring/Http/Diagrams/TimingsAggregator.cs

```diff
@@ -39,6 +44,23 @@ public IEnumerable<MonitoredEndpoint> AggregateIntoLogicalEndpoints()
                     CriticalTime = AggregateTimings(instanceCriticalTime)
                 };
 
+                var queueLength = queueLengths.FirstOrDefault(ql => ql.Key == endpointName);
+                var queueLengthValues = queueLength.Value?.OrderBy(kvp => kvp.Key).ToArray();
+
+                if (queueLengthValues != null)
```

Comment:
Good point. I don't think it is possible. The collection can be empty but not null.